tools/run.sh

```diff
@@ -47,7 +47,7 @@ done
 listen_for_devserver &
 
 # Run Celery worker process
-celery -A integreat_cms.integreat_celery worker -l INFO -B --concurrency=1 &
+deescalate_privileges celery -A integreat_cms.integreat_celery worker -l INFO -B --concurrency=1 &
```

Comment:
This is more a question than a comment, but: are we dependent on celery for the dev env? Because as far as I can tell, I can still do everything I normally would in the CMS when commenting out this line. I get that background workers won't be available, but will this crash the dev server, or just lead to errors in the logs?

If it is "optional", I would love the option to set an environment variable to skip starting celery (and thus not need redis installed).

If it is mandatory, I think the docs should mention that redis is also a mandatory prerequisite to running the dev server.